docs/team/dandaandaaaaaan.md

```diff
@@ -4,34 +4,43 @@ title: Daniel's Project Portfolio Page
 ---
 ### Project: ContacX
```

Comment:
Note the spelling of product name is wrong here.

Reply:
thanks for the spot, fixed in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/316/commits/7702cb4a1d86c0856f6ea62e77ccc7968cd115fc